### ISSUE TITLE
"Exit to Linux" is not enabled by default

### ISSUE THREAD

**Issue by Cheaterman** [opekope2/CosmicWineFixes#2]
Hi and thanks a lot for making this plugin, it works like a charm!

Sorry in advance because it's a bit of a dumb issue :sweat_smile: but I was simply scared for a moment because "Exit to Linux" isn't enabled by default despite what the README says, so I dug through the source code to see if eg you removed the feature, only to notice that it was configurable :-) (and again, when enabled, works like a charm indeed!)

Would you consider either fixing the README or switching the feature on by default? Thanks in advance :-)

EDIT: I opened a pull request in #3 that should switch the feature on by default :-)

**Comment by opekope2** [comment]
I do not mention any default options in the README All fixes/modifications are accessible from the settings screen, I have updated the README.
Also, I disabled it by default, since I don't think everyone would like such an option.
I also disabled "open log on crash", because that would probably annoy some people. It is a convenience functionality for those, who like to see why the game crashed (which is likely segfault/AccessViolationException).
Clipboard fix is enabled by default, as always has been (before the other two modifications).
